cmd/osquery-perf/mac10.14.6.tmpl

```diff
@@ -351,8 +351,14 @@
     "version": "{{ .Version }}",
     "type": "Application (macOS)",
     "bundle_identifier": "{{ .BundleIdentifier }}",
-    "source": "apps"
+    "source": "apps",
+    {{/* Note that in Go < 1.18, "and" and "or" functions evaluate all operands, so cannot be used for that */}}
```

Comment:
`or (and .LastOpenedAt .LastOpenedAt.Unix) ""`, which works on Go1.18 because they made `and` and `or` short-circuit evaluation.

Reply:
Might be good to include your answer in the comment. Without context, the current comment is a bit confusing.
```suggestion
    {{/* Note that in Go < 1.18, `{{ or (and .LastOpendedAt .LastOpenedAt.Unix) "" }}` won't work as expected because "and" and "or" don't short circuit. This was changed in Go 1.18 */}}
```